src/main/java/org/apache/skywalking/banyandb/v1/client/MeasureWrite.java

```diff
@@ -52,10 +51,6 @@ public MeasureWrite tag(String tagName, Serializable<BanyandbModel.TagValue> tag
         return (MeasureWrite) super.tag(tagName, tagValue);
     }
 
-    public MeasureWrite setID(String idValue) throws BanyanDBException {
-        return this.tag(Measure.ID, TagAndValue.idTagValue(idValue));
```

Comment:
Shall we remove ID type completely?

Reply:
Make sense. The ID type means nothing anymore.